src/main/java/io/forty11/web/Web.java

```diff
@@ -477,32 +544,11 @@ public static synchronized HttpClient getHttpClient() throws Exception
    {
       HttpClientBuilder b = HttpClientBuilder.create();
```

Comment:
I have another version of this change which also reuses httpclients (1 client per thread), but the payoff was smaller (though measurable) so I removed it.